app/config/settings_dev.py

```diff
@@ -53,3 +53,6 @@
     )
 
 SHELL_PLUS_POST_IMPORTS = ['from tests.data_factory import Factory']
+
+# Regex patterns of collections that should go to the managed bucket
+MANAGED_BUCKET_COLLECTION_PATTERNS = [r"^ch\.meteoschweiz\.ogd.*"]
```

Comment:
I'm unsure if we should have these in the environment. Or somewhere else.

Reply:
Environment makes sense to me, this way we can just change config without creating a new image